src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -26,7 +27,13 @@ internal static class NamedPipeUtils
 
         internal const string DefaultAppDomainName = "DefaultAppDomain";
         internal const string NamedPipeNamePrefix = "PSHost.";
+#if UNIX
+        internal const string NamedPipeNamePrefixSearch = "CoreFxPipe_PSHost*";
```

Comment:
If length is an issue, we could make this prefix smaller, and remove the domain name.